www/incubator/podlings/by-age.cgi

```diff
@@ -32,100 +35,82 @@ end
+          _circle r: 480, stroke: "#000", fill: "#000"
+          by_age.keys.sort.each do |age|
+            p1 = [Math.sin(theta)*475, -Math.cos(theta)*475].map(&:round).join(',')
+            theta += Math::PI*2 * by_age[age]/duration.length
+            p2 = [Math.sin(theta)*475, -Math.cos(theta)*475].map(&:round).join(',')
+            _path d: "M0,0 L#{p1} A475,475 0 0 1 #{p2} Z", 
+              fill: "##{colors[age]}", title: "#{by_age[age]} PMCs aged " +
+                "#{age} to #{age+1} year#{'s' if age>0}"
+          end
         end
+      }
+    ) do
+      # statistics
+      if duration.length % 2 == 0
+        mean = (duration[duration.length/2-1] + duration[duration.length/2])/2
+      else
+        mean = duration[duration.length/2]
       end
-    end
-
-    # pie chart
-    theta = 0
-    colors = ['0F0', 'FF0', 'F80', 'F50', 'F00', '800']
-
-    _svg_ viewBox: '-500 -500 1000 1000' do
-      _circle r: 480, stroke: "#000", fill: "#000"
-      by_age.keys.sort.each do |age|
-        p1 = [Math.sin(theta)*475, -Math.cos(theta)*475].map(&:round).join(',')
-        theta += Math::PI*2 * by_age[age]/duration.length
-        p2 = [Math.sin(theta)*475, -Math.cos(theta)*475].map(&:round).join(',')
-        _path d: "M0,0 L#{p1} A475,475 0 0 1 #{p2} Z", 
-          fill: "##{colors[age]}", title: "#{by_age[age]} PMCs aged " +
-            "#{age} to #{age+1} year#{'s' if age>0}"
-      end
-    end
-
-    # statistics
-    if duration.length % 2 == 0
-      mean = (duration[duration.length/2-1] + duration[duration.length/2])/2
-    else
-      mean = duration[duration.length/2]
-    end
-
-    _h2 'Statistics'
-    _p! do
-      _ "Count:      #{duration.length} PPMCs ("
-      _a 'history', href: 'https://incubator.apache.org/history/'
-      _ ")"
-    end
-    _p "Mean age:   #{(mean+0.5).to_i} days"
-    _p "Median age: #{(duration.reduce(:+)/duration.length + 0.5).to_i} days"
 
-    # Sorted list of podlings
-    _h2_! do
-      _a 'Currently in incubation',
-       href: 'https://incubator.apache.org/projects/#current'
-      _ ', sorted by age'
-    end
-
-    _table do
-      _tr do
-        table.at('tr').search('th').each do |th|
-          _th th.text
-        end
-      end
-      podlings.compact.sort.each do |date, tr|
-        _tr_ do
-          tr.search('td').each do |td|
-            a = td.at('a')
-            if a
-              _td! {_a a.text, href: (projects + a['href']).to_s}
-            else
-              _td td.text
+      _whimsy_panel_table(
+        title: 'Incubator Podlings By Age',
+        helpblock: -> {
+          _ul do
+            _li! do
+              _ "Count:      #{duration.length} PPMCs ("
+              _a 'history', href: 'https://incubator.apache.org/history/'
+              _ ') ('
+              _a 'source data', href: 'https://incubator.apache.org/projects/#current'
+              _ ')'
+            end
+            _li "Mean age:   #{(mean+0.5).to_i} days"
```

Comment:
Heh, cool.  
Re: features in the page: I just left all the data bits like the graph in the page, since I wasn't sure who used them.  If you want to change any of the content, go ahead.
Re: styles/layout: let me know when you want to get together to think about improvements to the common whimsy layout.  See:
https://github.com/apache/whimsy/blob/master/lib/whimsy/asf/themes.rb#L188